config/file_config.go

```diff
@@ -184,9 +184,17 @@ type RedisPeerManagementConfig struct {
 
 type CollectionConfig struct {
 	// CacheCapacity must be less than math.MaxInt32
-	CacheCapacity int        `yaml:"CacheCapacity" default:"10_000"`
-	MaxMemory     int        `yaml:"MaxMemory" default:"75"`
-	MaxAlloc      MemorySize `yaml:"MaxAlloc"`
+	CacheCapacity   int        `yaml:"CacheCapacity" default:"10_000"`
+	AvailableMemory MemorySize `yaml:"AvailableMemory" cmdenv:"AvailableMemory"`
+	MaxMemory       int        `yaml:"MaxMemory" default:"75"`
```

Comment:
My first thought reading this was that `MaxMemory` could also be a `MemorySize`, assuming that it represented the max amount of memory Refinery could use.  I didn't realize this is the max percentage of available memory Refinery can use.  Is there a way we can throw "percent"/"percentage" in the name somewhere?

Reply:
Fixed